### PR TITLE
Add SignUpNotifications screen for iOS users

### PR DESCRIPTION
**Pull request by bryanjos**
connects to #612

Screenshot:
Screen Shot 2019-03-13 at 12 29 25 PM

### CODE REVIEW

**Comment by bryanjos** [comment]
@brittanygay this should be ready for design updates.

The component is in `src/Auth/SignUpNotifications.js`

**Comment by brittanygay** [comment]
@bryanjos I just pushed up my styles for the iOS Notifications view. 

Assigning to @blazebarsamian for a design review and assigning back to you!

Screenshot:
Screen Shot 2019-03-13 at 12 29 25 PM

File: src/Auth/SignUpNotifications.js [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+    <Icon style={loginStyles.backButton} name="arrow-back" />
+  </TouchableHighlight>
+)
+
+export default class SignUpNotifications extends Component {
+  static propTypes = {
+    navigation: PropTypes.object.isRequired,
+  }
+
+  static navigationOptions = ({navigation}) => {
```
Comment (jwietelmann) [inline src/Auth/SignUpNotifications.js]: There's no reason for this to be a pre-bound lambda.

File: src/Auth/SignUpNotifications.js [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+/* eslint-disable camelcase,space-before-function-paren */
+
+const returnToButton = (navigation) => (
+  <TouchableHighlight
+    onPress={() => navigation.goBack()}
+    underlayColor="rgba(0, 0, 0, 0)"
+  >
+    <Icon style={loginStyles.backButton} name="arrow-back" />
+  </TouchableHighlight>
+)
```
Comment (jwietelmann) [inline src/Auth/SignUpNotifications.js]: Not a big fan of using lambdas without purpose. You lose the function name in the debugger. This isn't a particularly anonymous-feeling function. Also, seems like a React component to me.

File: src/Auth/SignUpNotifications.js [hunk 3]
```diff
@@ -0,0 +1,137 @@
...
+        style={loginStyles.container}
+        behavior="padding"
+        enabled
+      >
+        <ScrollView
+          showsVerticalScrollIndicator={false}
+          keyboardShouldPersistTaps={'handled'}
+        >
+          <View>
+            <Text style={[styles.headerSecondary, styles.textCenter, styles.paddingBottomLarge]}>Don't Miss Out!</Text>
```
Comment (jwietelmann) [inline src/Auth/SignUpNotifications.js]: I know it's stupid to request a change just to unbreak github diffs, but generally what we do here is
```
{`Don't Miss Out!`}
```
which takes care of the problem. github is bad with quotes in jsx string literals

File: src/Auth/SignUpNotifications.js (hunk 1 above)
Comment (bryanjos) [reply src/Auth/SignUpNotifications.js]: That's how the examples in the docs have it so I'm assuming there is a reason:
https://reactnavigation.org/docs/en/headers.html#overriding-shared-navigationoptions

Comment (jwietelmann) [reply src/Auth/SignUpNotifications.js]: I don't think there's a reason other than the docs author being a weirdo, but if you're scared to change it, I guess that's fine.

**Pull request merged**